t/01-wasm/directives/001-module_directive.t

```diff
@@ -136,7 +136,7 @@ qr/\[emerg\] .*? \[wasm\] open\(\) ".*?none\.wat" failed \(2: No such file or di
 --- no_error_log
 [error]
 [crit]
---- must_die
+--- must_die: 0
```

Comment:
Btw, we cannot go ahead with these changes. We *need* the master process to have a Wasm VM and detect invalid modules/bytecode, or else we start worker processes with an invalid configuration. This behavior is on purpose: it is so that we catch configuration errors and prevent Nginx from starting, just like configuration parsing also prevents Nginx from starting.

Reply:
I think the solution we should dig, which should also address other concerns and problems we have with the module, is that the master process should start a temporary VM that does all this, and then stop it before the `fork()`. Then forked worker processes restart the VM, assuming all configuration is valid (because checked by the master process).

This is actually the original intended design of this module ^. I've implemented all this maybe a couple of years ago, and there were changes with the V8 integration as well, I'm not sure how we stand with regards to this intended behavior today, and cannot check right moment.